server/index.js

```diff
@@ -1,66 +1,67 @@
-const express = require ('express');
-const bodyParser = require ('body-parser');
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+
 const db = require('./../database/');
+
 const PORT = '3000';
-const cors = require('cors')
+
 const app = express();
 app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({extended: true}));
-app.use(express.static(__dirname + '/../client/dist'));
-//Enable All CORS Requests
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static(`${__dirname}/../public/`));
+// Enable All CORS Requests
 app.use(cors());
 
-app.get('/products/similar', (req, res)=>{
-    db.SelectAllProduct(0, (err, result)=>{
-      if(err){
-        res.status(201).json({
-          error: err,
-          result: []
-        })
-      }else{
-        res.status(201).json({
-          error: null,
-          result: result
-        })
-      }
-    })
-})
+app.get('/products/similar', (req, res) => {
+  db.SelectAllProduct(0, (err, result) => {
+    if (err) {
+      res.status(201).json({
```

Comment:
201 status codes should be for successful `POST` requests. please send appropriate status codes. https://httpstatuses.com

Reply:
I 'll look back to this link today and put the right status code. You comment will help me improve my code skills and be more professional.